star/build_star_reference.py

```diff
@@ -0,0 +1,138 @@
+"""
+Builds a STAR reference genome index through Hail batch.
+"""
+import hailtop.batch as hb
+from cpg_utils import Path, to_path
+from cpg_utils.config import get_config
+from cpg_utils.hail_batch import remote_tmpdir, image_path, reference_path
+from textwrap import dedent
+
+# Get config and parameters
+config = get_config()
+BILLING_PROJECT = config['hail']['billing_project']
+DEFAULT_IMAGE = config['workflow']['driver_image']
+# IS_TEST = config['workflow']['access_level'] == 'test'
+TEST_BUCKET = config['storage']['common']['tmp']
+CPU = int(config['workflow'].get('n_cpu', 8))
+MEMORY = config['workflow'].get('memory', 'highmem')
+STORAGE = config['workflow'].get('storage', '150Gi')
+STAR_VERSION = str(config['star']['version'])
+SJDB_OVERHANG = int(config['star'].get('sjdb_overhang', 100))
+GENCODE_VERSION = str(config['star'].get('gencode_version', 44))
+GENCODE_BASE_URL = f'https://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_{GENCODE_VERSION}'
+GENCODE_GTF_BASENAME = f'gencode.v{GENCODE_VERSION}.primary_assembly.annotation.gtf'
+GENCODE_GTF_URL = f'{GENCODE_BASE_URL}/{GENCODE_GTF_BASENAME}.gz'
+# GENCODE_FASTA_BASENAME = 'GRCh38.primary_assembly.genome.fa'
+# GENCODE_FASTA_URL = f'{GENCODE_BASE_URL}/{GENCODE_FASTA_BASENAME}.gz'
+
+sb = hb.ServiceBackend(billing_project=BILLING_PROJECT, remote_tmpdir=remote_tmpdir())
+b = hb.Batch(backend=sb, default_image=DEFAULT_IMAGE)
```

Comment:
```suggestion
from cpg_utils import Path, to_path
from cpg_utils.config import get_config
from cpg_utils.hail_batch import image_path, reference_path, get_batch
from textwrap import dedent

# Get config and parameters
config = get_config()
# IS_TEST = config['workflow']['access_level'] == 'test'
TEST_BUCKET = config['storage']['common']['tmp']
CPU = int(config['workflow'].get('n_cpu', 8))
MEMORY = config['workflow'].get('memory', 'highmem')
STORAGE = config['workflow'].get('storage', '150Gi')
STAR_VERSION = str(config['star']['version'])
SJDB_OVERHANG = int(config['star'].get('sjdb_overhang', 100))
GENCODE_VERSION = str(config['star'].get('gencode_version', 44))
GENCODE_BASE_URL = f'https://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_{GENCODE_VERSION}'
GENCODE_GTF_BASENAME = f'gencode.v{GENCODE_VERSION}.primary_assembly.annotation.gtf'
GENCODE_GTF_URL = f'{GENCODE_BASE_URL}/{GENCODE_GTF_BASENAME}.gz'
# GENCODE_FASTA_BASENAME = 'GRCh38.primary_assembly.genome.fa'
# GENCODE_FASTA_URL = f'{GENCODE_BASE_URL}/{GENCODE_FASTA_BASENAME}.gz'

b = get_batch('Build Star Reference', default_image=config['workflow']['driver_image'])
```

I think this would be the change required for using the updated get_batch wrapper

Reply:
most of the stuff in the middle is unchanged, it's not the best way of presenting the change